src/MicrosoftTeams.ts

```diff
@@ -72,19 +74,34 @@ namespace microsoftTeams
         initializeCalled = true;
 
         // Undocumented field used to mock the window for unit tests
-        let currentWindow = this._window as Window || window;
+        currentWindow = this._window as Window || window;
+        let messageListener: (evt: MessageEvent) => void;
 
-        // If we are in an iframe then our parent window is the one hosting us (i.e. window.parent); otherwise,
-        // it's the window that opened us (i.e. window.opener)
+        // If we are in an iframe then our parent window is the one hosting us (i.e. window.parent)
         parentWindow = (currentWindow.parent !== currentWindow.self) ? currentWindow.parent : currentWindow.opener;
-        if (!parentWindow)
+        if (parentWindow)
         {
-            throw new Error("This page must be loaded in an iframe");
-        }
+            // Listen for messages post to our window (in a way that works for all browsers)
+            messageListener = (evt: MessageEvent) => processMessage(evt);
+            currentWindow.addEventListener("message", messageListener, false);
 
-        // Listen for messages post to our window (in a way that works for all browsers)
-        let messageListener = (evt: MessageEvent) => processMessage(evt);
-        currentWindow.addEventListener("message", messageListener, false);
+            try
+            {
+                // Send the initialized message to any origin since at this point we most likely don't know what our
+                // parent window's origin is yet and this message contains no data that could pose a security risk.
+                parentOrigin = "*";
+                let messageId = sendMessage("initialize");
+                callbacks[messageId] = (context: string, inDesktopClient: boolean) =>
```

Comment:
Yea, I guess I was just thinking we would make the client change first, but I guess since this is a versioned library, it doesn't really matter as long as the client changes happen before we actually release the new version. It does mean that we're deciding that the latest in master may not actually work against any current app, but I think that's probably fine since no one should be taking the latest (unversioned) bits especially since they wouldn't be on any CDN.